app/src/main/java/de/hhn/aib/labsw/blackmirror/dataclasses/MyMirror.kt

```diff
@@ -0,0 +1,47 @@
+package de.hhn.aib.labsw.blackmirror.dataclasses
+
+import java.io.Serializable
+import java.util.ArrayList
+
+/**
+ * This class represents the mirror.
+ *
+ * @author Niklas Binder
+ * @version 24-05-2022
+ */
+class MyMirror : Serializable {
```

Comment:
Why isn't this class simply called `Mirror`?

Reply:
also perhaps this class should/could be a data class (see https://kotlinlang.org/docs/data-classes.html)